index.html

```diff
@@ -0,0 +1,848 @@
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>TNW</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="icon"
+      type="image/png"
+      href="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/icons/favicon-32x32.png?v=1589297875"
+      sizes="32x32"
+    />
+    <link rel="stylesheet" href="assets/css/reset.css" />
+    <link rel="stylesheet" href="assets/css/main.css" />
+    <link rel="stylesheet" href="assets/css/font.css" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link
+      href="https://kit-free.fontawesome.com/releases/latest/css/free.min.css"
+      media="all"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <nav class="main-nav">
+      <div class="nav-logo">
+        <a href="#">
+          <img src="./assets/img/tnw.svg" alt="The next Logo" />
+        </a>
+      </div>
+      <div class="nav-top-desktop">
+        <ul>
+          <li><a href="#">News</a></li>
+          <li><a href="#">Events</a></li>
+          <li><a href="#">Business</a></li>
+          <li><a href="#">AMAs</a></li>
+          <li><a href="#">Spaces</a></li>
+          <li><a href="#">Terms & Conditions</a></li>
+        </ul>
+        <ul>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Advertise</a></li>
+          <li><a href="#">Jobs</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </div>
+      <div class="nav-icons">
+        <ul>
+          <li><a href="">Latest</a></li>
+          <li><a href="">Hard Fork</a></li>
+          <li><a href="">Plugged</a></li>
+          <li><a href="">Readme</a></li>
+          <li><a href="">Growth Quarters</a></li>
+        </ul>
+        <ul class="nav-icons-list">
+          <li>
+            <a href="#">
+              <span class="material-icons">search</span>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <span class="material-icons">menu</span>
+            </a>
+          </li>

```

Comment:
- [ ] The menu icon or hamburguer should only apper for the smallest screen.

> Your page
![image](https://user-images.githubusercontent.com/57414789/82391415-d9a26000-9a06-11ea-852d-5e5184f2906c.png)

> Original
![image](https://user-images.githubusercontent.com/57414789/82391450-ecb53000-9a06-11ea-972a-7acb2883f197.png)

Reply:
Hi Lucas ! We could change this quickly, could we do it now and it will pass ?

Only forgot to keep the display: none in one of the queries !

Reply:
Asked for it via Slack. We have to re-submit for a **new review.**

Thank you Lucas for taking the time and responding through Slack!

@simandebvu I've already made the changes so the next try should pass!